jekyll/_cci2/images/linux-vm/16.04-to-20.04-migration.md

```diff
@@ -0,0 +1,117 @@
+---
+layout: classic-docs
+title: "Migrating from Ubuntu 16.04 to Ubuntu 20.04"
+short-title: "16.04 to 20.04 Migration"
+description: |
+  Helpful information to aid in migrating from the CircleCI Ubuntu 16.04 images
+  to the CircleCI Ubuntu 20.04 images. This is less of a step-by-step guide but
+  rather one that points to changes and common gotchas.
+order: 20
+version:
+- Cloud
+- Server v3.x
+- Server v2.x
+---
+
+* TOC
+{:toc}
+
+## Switching to a supported image
+{: #switching-to-a-supported-image }
+
+Swapping out the deprecated image for a supported one in the CircleCI config is straight forward.
+Additional work to get your pipelines to run with the newer image will depend on if the [changes below](#changes) apply to your project.
+Here's an example config using an Ubuntu 16.04 image vs one using a 20.04 image:
+
+{:.tab.linuxVM.Deprecated-Image}
+```yaml
+version: 2.1
+jobs:
+  my-job:
+    machine:
+      image: ubuntu-1604:202007-01
+    steps:
+      run: echo "do something"
+```
+
+{:.tab.linuxVM.Supported-Image}
+```yaml
+version: 2.1
+jobs:
+  my-job:
+    machine:
+      image: ubuntu-2004:202201-02
+    steps:
+      run: echo "do something"
+```
+
+Supported images (and tags) can be found in the [Developer Hub](https://circleci.com/developer/images?imageType=machine).
+
+
+## Changes
+{: #changes }
+
+There are many changes between Ubuntu 16.04 and 20.04 that you should be aware of before migrating.
+Many are changes that Canonical has made (the company behind Ubuntu), but also changes that CircleCI has made to the software that we pre-install.
+Let's go through each of these changes separately.
+
+### Changes to Ubuntu
```

Comment:
Just to confirm - this is a final list of changes by Canonical, right? (Meaning that we won't have to keep updating this list?) I am wondering if this might be a good candidate for a snippet since this info is basically the same on both 14.04 -> 20.04 and 16.04 -> 20.04 guides (unless my eyes missed some things)